themes/src/main/resources/theme/base/admin/resources/partials/user-credentials.html

```diff
@@ -97,7 +97,7 @@
                 <div class="form-group">
                     <label class="col-md-2 control-label" for="confirmPas">{{:: 'password-confirmation' | translate}} <span class="required" data-ng-show="create">*</span></label>
                     <div class="col-md-6">
-                        <input class="form-control" kc-password id="confirmPas" name="confirmPas" data-ng-model="confirmPassword" required>
+                        <input class="form-control" kc-password type="text" id="confirmPas" name="confirmPas" data-ng-model="confirmPassword" required>
```

Comment:
If this change is needed then there's a lot of other places that use kc-password that needs to be updated as well. Might be better to just make the directive do it rather than change the html directly though?

Reply:
This change was accidental, it's not needed